packages/ui/src/components/va-stepper/types.ts

```diff
@@ -2,6 +2,9 @@ export type Step = {
   label: string
   icon?: string
   disabled?: boolean
+  save?: Function
```

Comment:
```suggestion
  save?: (step: Omit<Step, 'save'>) => boolean
```

Let's specify function type. Let's say save first argument is going to be a step, so we can do:

```
save(step) {
  if (validate()) {
    step.complete = true
  } else {
    step.hasError = true
  }
}
```

I also omited save method, so user will not be able to call save in save and in save. (There will be TS error, actually he can)

Reply:
I did remove the form validation from the stepper its self. Is there an additional change I need to make?